commands/info.md

```diff
@@ -573,6 +573,7 @@ Here is the meaning of all fields in the **replication** section:
 
 *   `role`: Value is "master" if the instance is replica of no one, or "slave" if the instance is a replica of some primary instance.
      Note that a replica can be primary of another replica (chained replication).
+*   `replicas_waiting_psync`: The number of replicas waiting for psync during the dual-channel replication. Added in Valkey 8.0.
```

Comment:
Spellcheck complains about "psync". It is not a normal word. In other parts of the documentation we say "partial resynchronization" or "partial resync".

```suggestion
*   `replicas_waiting_psync`: The number of replicas waiting for partial resynchronization during the dual-channel replication. Added in Valkey 8.0.
```